backend/graphql/types/userType.ts

```diff
@@ -41,6 +51,7 @@ const userType = gql`
     users: [UserDTO!]!
     usersByRole(role: String!): [UserDTO!]!
     usersCSV: String!
+    teachers: [TeacherDTO]
```

Comment:
```suggestion
    teachers: [TeacherDTO!]!
```
assuming we always have teacher users